src/main/java/org/asamk/signal/Manager.java

```diff
@@ -775,6 +775,59 @@ public void sendEndSessionMessage(List<String> recipients) throws IOException, E
         sendMessage(messageBuilder, recipients);
     }
 
+    @Override
+    public String getContactName(String number) {
+        ContactInfo contact = contactStore.getContact(number);
+        if(contact == null) {
```

Comment:
Please add a space after the if -> "if ("
also for the other ifs added in this PR.